### PR TITLE
5 login function

### PR DESCRIPTION
**Pull request by StephenZeiler**

### CODE REVIEW

File: .github/workflows/azure-static-web-apps-yellow-sea-0fa5a340f.yml [hunk 1]
```diff
@@ -10,6 +10,9 @@ on:
     branches:
       - main
 
+env:
+  CI: false
```
Comment (apla1297) [inline .github/workflows/azure-static-web-apps-yellow-sea-0fa5a340f.yml]: Why are you making this false?

File: .github/workflows/main_locatecontractorapi.yml [hunk 2]
```diff
@@ -8,7 +8,10 @@ on:
 
     branches:
       - main
-  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - main
```
Comment (apla1297) [inline .github/workflows/main_locatecontractorapi.yml]: This is going to make your deployments happen on any PR action, you only want your app to close when a PR is merged into main.

File: react-locate-contractor/public/index.html [hunk 3]
```diff
@@ -10,34 +10,14 @@
...
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
-
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
```
Comment (apla1297) [inline react-locate-contractor/public/index.html]: This line is somewhat concerning from a security standpoint

File: API-LocateContractor/obj/Debug/net6.0/project.razor.json [hunk 4]
```diff
@@ -12,7 +12,7 @@
   "ProjectWorkspaceState": {
```
Comment (apla1297) [inline API-LocateContractor/obj/Debug/net6.0/project.razor.json]: Seems your debug and bin folders a sneaking back in. Check your ignore and make sure your file tracking is clean

Comment (StephenZeiler) [reply API-LocateContractor/obj/Debug/net6.0/project.razor.json]: I think I was able to take care of this.

File: .github/workflows/main_locatecontractorapi.yml (hunk 2 above)
Comment (StephenZeiler) [reply .github/workflows/main_locatecontractorapi.yml]: I took out everything in front of type except "closed", is this what you mean by close the app on merge to main?

File: .gitignore [hunk 5]
```diff
@@ -21,3 +22,19 @@
...
 yarn-error.log*
+node_modules
+bin
+.DS_*
+*.log
+logs
+**/*.backup.*
+**/*.back.*
+
+node_modules
```
Comment (Krescenskok) [inline .gitignore]: Is there a reason why you have node_modules on line 25 as well?

File: react-locate-contractor/src/Pages/Login.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+    const onError = (err: any) => {
+        console.log('Sign in error:', err);
+    };
+
+    if (!authState) {
+        return <div>Loading ... </div>;
+    }
+
+    console.log(config)
```
Comment (Krescenskok) [inline react-locate-contractor/src/Pages/Login.tsx]: console.log should be removed

**Review by Krescenskok — CHANGES_REQUESTED** [review]

**Review by apla1297 — COMMENTED** [review]
Overall I think this can move once the merge conflicts are resolved and the comments from Krescens are resolved.

File: .gitignore (hunk 5 above)
Comment (StephenZeiler) [reply .gitignore]: It was a mistake brought in from the template I used. I removed it.

File: react-locate-contractor/src/Pages/Login.tsx (hunk 6 above)
Comment (StephenZeiler) [reply react-locate-contractor/src/Pages/Login.tsx]: I removed it.

**Review by Krescenskok — APPROVED** [review]

**Pull request merged**